tokio-postgres/src/config.rs

```diff
@@ -534,6 +557,12 @@ impl fmt::Debug for Config {
             }
         }
 
+        let replication_mode_str = match self.replication_mode {
```

Comment:
Why not just derive debug for replication mode?

Reply:
fixed